datafusion/core/tests/sql/arrow_typeof.rs

```diff
@@ -1,139 +0,0 @@
-// Licensed to the Apache Software Foundation (ASF) under one
-// or more contributor license agreements.  See the NOTICE file
-// distributed with this work for additional information
-// regarding copyright ownership.  The ASF licenses this file
-// to you under the Apache License, Version 2.0 (the
-// "License"); you may not use this file except in compliance
-// with the License.  You may obtain a copy of the License at
-//
-//   http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing,
-// software distributed under the License is distributed on an
-// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
-// KIND, either express or implied.  See the License for the
-// specific language governing permissions and limitations
-// under the License.
-
-use super::*;
-
-#[tokio::test]
-async fn arrow_typeof_null() -> Result<()> {
-    let ctx = SessionContext::new();
-    let sql = "SELECT arrow_typeof(null)";
-    let actual = execute(&ctx, sql).await;
-    let expected = "Null";
-    assert_eq!(expected, &actual[0][0]);
-
-    Ok(())
-}
-
-#[tokio::test]
-async fn arrow_typeof_boolean() -> Result<()> {
```

Comment:
I double checked -- here is the corresponding .slt file: 

https://github.com/apache/arrow-datafusion/blob/dcd52ee3d87c4dd9e2c176165e9e20644f66988b/datafusion/core/tests/sqllogictests/test_files/arrow_typeof.slt#L26